webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -21,7 +21,7 @@ class NotificationDrawerToggle extends Component {
     switch (actionType) {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({
-          count: NotificationsStore.getNotifications().length,
+          unreadNotificationsCount: NotificationsStore.hasUnreadNotifications(),
```

Comment:
calling NotificationsStore.hasUnreadNotifications without arguments
method signature is `hasUnreadNotifications(notifications)`
note also that this only checks unread notifications when notifications are received. Consider the case when there is one unread notification and it is manually marked as read. Then there are no unread notifications but state won't reflect that until next polling request